Lib/test/test_tabnanny.py

```diff
@@ -233,8 +234,12 @@ def test_when_nannynag_error(self):
     def test_when_no_file(self):
         """A python file which does not exist actually in system."""
         path = 'no_file.py'
-        err = f"{path!r}: I/O Error: [Errno {errno.ENOENT}] " \
-              f"No such file or directory: {path!r}\n"
+        if sys.platform == "vxworks":
+            err = f"{path!r}: I/O Error: [Errno {errno.ENOENT}] " \
+                  f"no such file or directory: {path!r}\n"
+        else:
+            err = f"{path!r}: I/O Error: [Errno {errno.ENOENT}] " \
+                  f"No such file or directory: {path!r}\n"
```

Comment:
Can you please to use os.strerror(errno.ENOENT) to avoid the if? Something like:

```
            err = (f"{path!r}: I/O Error: [Errno {errno.ENOENT}] "
                  f"{os.strerror(errno.ENOENT)}: {path!r}\n")
```

Note: I would prefer to use parenthesis to avoid trailing "\"  ;-))